### PR TITLE
Moves gun calibers to defines

### LINKED ISSUE #18392: Projectile weaponry does not use defines for caliber
Right now it uses just strings, such as caliber = "9mm"

Would be a lot easier if they used defines.

### PR DESCRIPTION
**Pull request by MadmanMartian**
Might help in the future, should somebody want to add another 9mm handgun for instance.

Closes #18392

### CODE REVIEW

**Comment by DamianX** [comment]
A bit too generic for my taste, I suggest you give them some prefix or suffix. Like `SHOTGUN` which could mean anything.

**Comment by DamianX** [comment]
i guess

File: __DEFINES/weapons.dm [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+#define POINT762 "a762"
+#define POINT762X55 "7.62x55"
+#define BROWNING50 ".50BMG"
+
+#define ROCKETGRENADE "rpg"
+#define GUIDEDROCKET "guided rocket"
+
+
+#define PULSE "pulse"
+#define GAUGE12 "12 guage"
```
Comment (Intigracy) [inline __DEFINES/weapons.dm]: :thinking:

Comment (MadmanMartian) [reply __DEFINES/weapons.dm]: Nice spot

**Pull request merged**